### DISCUSSION TITLE
Frozen Colums overlaping

### DISCUSSION THREAD

**Discussion by juniorHlatshwayo** [anvilistas/tabulator#92] [Q&A] [answered]
Hi, I am having an issue with how my table displays after making columns "frozen". When I scroll horizontally, the frozen columns overlap each other instead of all frozen columns being visble. Here is my code and here is a screen shot of the output.

`self.tabulator.columns = [ row_selection_column,
            {"title": "Name", "field": "name", "formatter": "link","frozen":True, "width":200},
            {"title": "Surname", "field": "surname","frozen"![Screenshot 2023-01-18 100124](https://user-images.githubusercontent.com/33415818/213115932-7d63f8db-de9b-41a9-8149-c43ae462b44d.png)True, "formatter": "link", "width":200},
            {"title": "Sex", "field": "sex", "width":200},
            {"title": "Email", "field": "email", "formatter": "link", "width":200},
            {"title": "Number", "field": "number", "formatter": "link", "sorter": "number","width":200},
        ]`

![Screenshot 2023-01-18 100124](https://user-images.githubusercontent.com/33415818/213116100-f9e6112a-15d2-4bee-9f32-7711fa346657.png)

**Comment by s-cork** [comment, marked as answer]
Looks like you're coming up against a change in Tabulator version.
I'll make sure to adjust that in the docs

try `"position": "sticky"` instead and let me know how it goes.

**Reply by juniorHlatshwayo** [reply]
I tried it and now I get an error:
`AttributeError: The following column option(s) are invalid: 'position'. You may need to include the required Module in Tabulator.modules.
at app/tabulator/Tabulator/_custom_modules.py:314`

I tried to include in the Module as well but still not working.

**Reply by s-cork** [reply]
oh - i see the issue - you're using the `row_selection_column`
But that doesn't have `frozen` set to `True`

Do:

```python
row_selection_column["frozen"] = True

```

at some point before the column definition

**Reply by juniorHlatshwayo** [reply]
Working perfectly, thank you so much
